### DISCUSSION TITLE
vcoco pretrained model

### DISCUSSION THREAD

**Discussion by learn2phoenix** [fredzzhang/spatio-attentive-graphs#62] [Q&A] [answered]
have you released the weights file for vcoco model? If so, could you please point me to that?

**Comment by fredzzhang** [comment, marked as answer]
Hi, @learn2phoenix 

We did not release the weights for the V-COCO model because it is all reproducible with the code. Training the model on V-COCO only takes a small amount of time compared to the HICO-DET model. So we only released the model on HICO-DET.

Regards,
Fred.
